converter/src/delphesHelpers.h

```diff
@@ -28,18 +29,37 @@ inline LorentzVectorT getP4(const T& particle) {
   };
 }
 
+namespace {
 /**
- * Compare two 4-momentum vectors to be within a relative tolerance in the
- * energy and all momentum components. The energy requirement can be dropped on
- * demand.
+ * Combination of a relative and absolute comparison. The absolute comparison
+ * should take precedence if the compared values are small, where relative
+ * differences can become very large. The definition of "small" depends on the
+ * ratio of relEps / absEps. E.g. for equal values everything for |a|, |b| < 1
+ * is considered small. In general max(|a|, |b|) < absEps / relEps defines where
+ * "small" begins for inputs a and b
+ */
+  bool cmp_float(float a, float b, float absEps=1e-6, float relEps=1e-6) {
```

Comment:
In some float comparisons (for example numpy.isclose) the abolute and relative tolerance are added and then compared to the difference. I think this is fine too, though.

Reply:
That is interesting to know, I actually haven't checked the numpy implementation. In any case, I think this should be a much more robust and valid float comparison implementation than the previous one, which was failing quite often, especially for small numbers.